src/electron/views/automated-checks/components/command-bar.tsx

```diff
@@ -36,9 +36,7 @@ export const CommandBar = NamedFC<CommandBarProps>('CommandBar', (props: Command
         disabled: props.scanStoreData.status === ScanStatus.Scanning,
     };
 
-    // UICommandBar doesn't do null checks for these. Use empty lists for "no items", not null/undefined.
     const items: ICommandBarItemProps[] = [rescanCommandBarItem];
-    const farItems: ICommandBarItemProps[] = [];
 
-    return <UICommandBar items={items} farItems={farItems} className={styles.commandBar} />;
+    return <UICommandBar items={items} className={styles.commandBar} />;
```

Comment:
**NOTE**: Passing a `farItems` property was necessary as of office fabric 6.x.y. 

Using an empty array for the `farItems` property cause this component to fail automated checks; `farItems` is use to create an element with `role="menubar"` but, since there are no items, there is no child with `role="menuitem"` which makes **aria-required-children** rule to fail.